plugins/woocommerce-blocks/package.json

```diff
@@ -395,6 +396,15 @@
 			"command": "webpack --watch",
 			"service": true
 		},
+		"build:project:blocks-json-to-php": {
+			"command": "node ./bin/blocks-json-to-php.js",
```

Comment:
Where is this script? I assume it's in https://github.com/woocommerce/woocommerce/pull/50889?

Reply:
Ah oops, I forgot this script in this one. Now included.